### PR TITLE
Added fail2ban

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

**Comment by hy0shida** [comment]
Do we want to modify /etc/fail2ban/jail.conf rather than providing customizations in /etc/fail2ban/jail.local?

**Comment by himorin** [comment]
I wondered whether is better to update config file or to place a file in jail.d and overriding the defaults. 
Considering mis-operation at local which someone could do, it is unsafe to configure using override which could cause a fallback to the defaults.

File: roles/fail2ban/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+- name: restart fail2ban
+  systemd:
+    name: fail2ban.service
+    state: restarted
+    daemon_reload: yes
+    enabled: yes
+  become: yes
```
Comment (yukimoritani) [inline roles/fail2ban/handlers/main.yml]: I'm afraid this line should be before the line 3 to become root.
Actually I failed to restart fail2ban, while by setting this line before the line 3, I succeeded.

File: README.rst [hunk 2]
```diff
@@ -23,6 +23,8 @@ Roles
   Install of eups, and its required python packages
 :exim4:
   Configuration of exim4 as satellite host
+:fail2ban:
+  Install and configure fail2ban package (require option 'jail_enable')
```
Comment (yukimoritani) [inline README.rst]: For clarifying, to have options, could you write an example in site.yml or somewhere? 
Or Are you planning to write something at some point?
I'm sorry.... I'm not so bright to your plan to make site.yml for operating ansible...

Comment (himorin) [reply README.rst]: Such document should go into docs/ ones, once PR #27 is finished, I think.

File: roles/fail2ban/handlers/main.yml (hunk 1 above)
Comment (himorin) [reply roles/fail2ban/handlers/main.yml]: Order in dict of yaml configuration should not matter, I think. (at least, other roles have similar order.)
Is it an error from ansible role, or from mis-input of sudo password?

**Pull request merged**